timesheets/README.md

```diff
@@ -0,0 +1,25 @@
+# Manage employee time sheets 
+
+Allows employers to view their employees' weekly timesheets, easily calculate their weekly pay, and send automated emails notifying them of timesheet approval/disapproval status.
+
+## Description
+
+Quickly create a hands-free employee pay management system. Start with collecting employees' timesheets in a Google Form, then use Apps Script to create a spreadsheet to easily view, compile, and manage their data. After everybody submits their responses, calculate their pay, approve or disapprove their pay, and auto-send emails notifying them of their pay status.
+
+## Set-Up Instructions
+
+1. Create a copy of the sample [Employee Weekly TimeSheet Form](***LINK***).
+1. Once you've received responses, on your google form hit **Edit > View Summary***. Click the three summary dots in the top right corner, and click on **Select Response Destination** from the drop-down.
+1. Create a new Google Sheet and name it whatever you'd like. The Form responses will now appear in this sheet.
+1. On your Google Sheet, go to **Tools > Script Editor**
```

Comment:
Having to copy over the script code isn't ideal. Would it be possible to have a spreadsheet with the script attached that the user can copy, and then link the copied form to the copied spreadsheet?

Reply:
Yes ! I did not know this was possible, but will do !